ajax/libs/3Dmol/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "3Dmol",
+  "filename": "3Dmol-min.js",
+  "version": "1.0.6",
+  "description": "Object oriented Javascript molecular visualization library",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/3dmol/3Dmol.js.git"
+  },
+  "keywords": [
+    "3Dmol.js",
+    "webGL"
+  ],
+  "author": "David Koes and contributors",
+  "homepage": "http://3dmol.csb.pitt.edu",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/3dmol/3Dmol.js.git",
+    "basePath": "release",
+    "files": [
+      "**/*"
+    ]
+  },
+  "licenses": [
+    "MIT",
+    "LGPL-3.0",
+    "BSD-3-Clause"
```

Comment:
[L1](https://github.com/3dmol/3Dmol.js/blob/master/LICENSE#L1) in `LICENSE` file show __3Dmol.js incorporates code from GLmol__ and [L37](https://github.com/3dmol/3Dmol.js/blob/master/LICENSE#L37) said __GLmol under dual license of MIT or LGPL3__.  

So, do they seperate? I mean that `3Dmol.js` is under BSD-3-Clause only and `GLmol` is under MIT or LGPL3.